### PR TITLE
#3933 - Add support for order comments in graphql

### PR DESCRIPTION
**Pull request by IrinaZhadzinets**

### CODE REVIEW

File: Model/Formatter/Order.php [hunk 1]
```diff
@@ -164,4 +165,25 @@ public function isRssAllowed()
         );
     }
 
+    /**
+     * Get order comments in proper format
+     *
+     * @param OrderInterface $orderModel
+     * @return array
+     */
+    protected function getOrderComments(OrderInterface $orderModel): array
```
Comment (aleksandrsm) [inline Model/Formatter/Order.php]: would be nice if it was a public function

**Review by aleksandrsm — APPROVED** [review]

File: Model/Formatter/Order.php [hunk 2]
```diff
@@ -164,4 +165,25 @@ public function isRssAllowed()
...
+        $comments = [];
+
+        foreach ($orderModel->getStatusHistoryCollection() as $comment) {
+            if ($comment->getIsVisibleOnFront()) {
+                $comments[] = [
+                    'timestamp' => $comment->getCreatedAt(),
+                    'message' => $comment->getComment()
+                ];
+            }
+        }
```
Comment (AleksandrsKondratjevs) [inline Model/Formatter/Order.php:185]: So there was a graphql endpoint for invoice comments but resolver was never returning them from default magento?)

File: Model/Resolver/Invoices.php [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+     *
+     * @param InvoiceInterface $invoice
+     * @return array
+     */
+    public function getInvoiceComments(InvoiceInterface $invoice): array
+    {
+        $comments = [];
+
+        foreach ($invoice->getComments() as $comment) {
+            if ($comment->getIsVisibleOnFront()) {
```
Comment (AleksandrsKondratjevs) [inline Model/Resolver/Invoices.php]: So there was a graphql endpoint for invoice comments but resolver was never returning them from default magento?)

Comment (IrinaZhadzinets) [reply Model/Resolver/Invoices.php]: Well, at least judging by my local magento files, it is...

File: Model/Resolver/Invoices.php [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+        foreach ($orderModel->getInvoiceCollection() as $invoice) {
+            $invoices[] = [
+                'id' => base64_encode($invoice->getEntityId()),
+                'number' => $invoice['increment_id'],
+                'comments' => $this->getInvoiceComments($invoice),
+                'model' => $invoice,
+                'order' => $orderModel
+            ];
+        }
+        return $invoices;
```
Comment (AleksandrsKondratjevs) [inline Model/Resolver/Invoices.php]: Add please white-space before return

**Review by AleksandrsKondratjevs — APPROVED** [review]

File: Model/Resolver/Invoices.php (hunk 3 above)
Comment (AleksandrsKondratjevs) [reply Model/Resolver/Invoices.php]: Yea, also checked, magento created endpoint in graphql, but it is always null)

**Pull request merged**